plugins/rule_engines/irods_rule_language/src/functions.cpp

```diff
@@ -1983,6 +1983,14 @@ Res *smsi_remoteExec( Node** paramsr, int, Node* node, ruleExecInfo_t* rei, int,
 /// 
 /// \return integer
 /// \retval 0 upon success
+///
+/// \b Example
```

Comment:
Should we put a `\usage` tag here or something? I'm not saying this is wrong, but it's just kind of floating out here so I'm not really sure how it's going to appear in the rendering. I don't know enough about Doxygen. :)

Reply:
The new APIs use that. I can't remember why I chose to use `\b Example` over `\usage`. It may be due to flexibility or better control over the rendering.

Reply:
Or maybe I didn't like how `\usage` affected the rendering.

Happy to discuss if you and @JustinKyleJames want to. I can render the docs quite easily.

Reply:
Yeah, I don't feel too strongly about this. It does seem that using `\usage` results in the string "Example Usage:" for other Doxygen renderings.

Reply:
Ok. For now, let's leave it as is. We can adjust later on if needed.